doc/examples/applications/plot_3d_structure_tensor.py

```diff
@@ -53,7 +53,7 @@
     data[n_plane // 2, :, :, 1],
     zmin=v_min,
     zmax=v_max,
-    labels={'x': 'Y', 'y': 'X', 'color': 'intensity'}
+    labels={'x': 'row', 'y': 'col', 'color': 'intensity'}
```

Comment:
it's x=col, y=row. =)

Reply:
Aww there we go!! It's because x=col and y=row that I loaded the dataset with the following dimension order: [z, y, x, c] = [plane, row, column, channel] :relieved: 

Ok, so I'll revert 58eec1cea6afa66c134cc19097d3ec7c0f39459d (where I introduced the inversion) and proceed with the expected renaming :sweat_smile: